pubspec.yaml

```diff
@@ -6,7 +6,7 @@ repository: https://github.com/jonbhanson/flutter_native_splash
 issue_tracker: https://github.com/jonbhanson/flutter_native_splash/issues
 
 environment:
-  sdk: '>=2.18.0 <3.0.0'
+  sdk: '>=2.19.0 <3.0.0'
```

Comment:
I had to increase the minimum Dark SDK value because of the Isolate.run() method.
More info here: https://api.dart.dev/dev/3.1.0-126.0.dev/dart-isolate/Isolate/run.html